### ISSUE TITLE
Trait IntoRustByName<Uuid> not implemented for Row

### ISSUE THREAD

**Issue by tinrab** [AlexPikalov/cdrs#206]
I'm trying to select an UUID field as shown in this [example](https://github.com/AlexPikalov/cdrs/blob/master/examples/all.rs#L876), using the same crate version for **uuid**:

```rust
let _: Uuid = row.r_by_name("id").expect("my_uuid");
```

But I get this error:

```
error[E0277]: the trait bound `cdrs::types::rows::Row: cdrs::types::IntoRustByName<uuid::Uuid>` is not satisfied
   --> src\db.rs:106:25
    |
106 |       let _: Uuid = row.r_by_name("id").expect("my_uuid");
    |                         ^^^^^^^^^ the trait `cdrs::types::IntoRustByName<uuid::Uuid>` is not implemented for `cdrs::types::rows::Row`
    |
    = help: the following implementations were found:
              <cdrs::types::rows::Row as cdrs::types::IntoRustByName<f64>>
              <cdrs::types::rows::Row as cdrs::types::IntoRustByName<std::net::IpAddr>>
              <cdrs::types::rows::Row as cdrs::types::IntoRustByName<uuid::Uuid>>
              <cdrs::types::rows::Row as cdrs::types::IntoRustByName<cdrs::types::map::Map>>
            and 12 others
```

My toolchain:

```
nightly-x86_64-pc-windows-msvc (default)
rustc 1.27.0-nightly (ac3c2288f 2018-04-18)
```

**Comment by AlexPikalov** [comment]
@tinrab 
I think you may need to change:

```rust
let _: Uuid = row.r_by_name("id").expect("my_uuid");
```
to
```rust
let _: uuid::Uuid = row.r_by_name("id").expect("my_uuid");
```
But to make sure could you please past a your "imports"? 

BTW: does simple `cargo run --example all` work on your configuration?

**Comment by AlexPikalov** [comment]
I was able to reproduce it locally. Investigating.

Thanks for reporting

**Comment by AlexPikalov** [comment]
@tinrab Despite CDRS declares UUID `0.6.1` in `Cargo.toml` as a dependency crate itself requires `^0.5.1` version of UUID (see Dependencies section in https://crates.io/crates/cdrs/1.2.2), so as a quick fix I suggest to put `uuid = "0.5"` into your crate `Cargo.toml`.

And I'll try to provide a fix ASAP.

**Comment by AlexPikalov** [comment]
@tinrab Fix has been published. Please update your CDRS dependency to `cdrs="1.2.3"` and use UUID version that CDRS is depended upon (`0.6.3`)

Please feel free to create an issues if the problem persists
